### PR TITLE
Add Organisation About Page 

### PR DESCRIPTION
**Pull request by davidgisbey**
## Description

We're moving the about us information to it's own tab for organisations. This adds the tab to the helper method responsible for rendering the organisation tabs and then adds a new endpoint which handles rendering the about page

image

## Trello card

https://trello.com/c/HW4NnOjF/348-update-organisations-details-page-and-add-an-about-page

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

### CODE REVIEW

**Comment by davidtrussler** [comment]
@davidgisbey @nnagewad Just an observation when looking at this PR: if I go to the "About" tab of an organisation that doesn't have such a thing I get the message "No about us page.", which is fine but maybe it's not clear to a user in this scenario how they would create an "About Us" page. Should we think about adding a link to the page where this is done?

**Comment by nnagewad** [comment]
@davidtrussler maybe that can be a separate ticket 👍🏽

File: app/views/admin/organisations_about/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+<% content_for :context, organisation_context_block(current_user, @organisation)  %>
+<% content_for :title_margin_bottom, 4 %>
+
+<p class="govuk-body">
+  <%= view_on_website_link_for @organisation, class: "govuk-link", target: "blank" %>
+</p>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "components/secondary_navigation", {
```
Comment (davidtrussler) [inline app/views/admin/organisations_about/show.html.erb:12]: The `secondary_navigation` component needs to be within a full-width container - in this case the tabs overflow the container.

Comment (davidgisbey) [reply app/views/admin/organisations_about/show.html.erb]: great catch have sorted thanks

**Review by davidtrussler — APPROVED** [review]
👍

**Pull request merged**